docs/TELEMETRY_SPECIFICATION.md

```
@@ -0,0 +1,1167 @@
+All components MUST use [W3C Trace Context](https://www.w3.org/TR/trace-context/) for context propagation.
+
+#### HTTP Headers
+
+| Header | Required | Description |
+|--------|----------|-------------|
+| `traceparent` | YES | Contains trace-id, parent-id, and trace-flags |
+| `tracestate` | NO | Vendor-specific trace data |
+
+#### traceparent Format
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)